src/OrchardCore.Modules/OrchardCore.Demo/Views/Todo.Edit.liquid

```diff
@@ -46,5 +46,5 @@
     </table>
 {% endform %}
 
-{% script  src:"https://cdn.jsdelivr.net/npm/jquery-validation@1.19.1/dist/jquery.validate.min.js", at: "Foot", dependsOn: "jquery" %}
-{% script  src:"https://cdn.jsdelivr.net/npm/jquery-validation-unobtrusive@3.2.11/dist/jquery.validate.unobtrusive.min.js", at: "Foot", dependsOn: "jquery" %}
+{% script src:"https://cdn.jsdelivr.net/npm/jquery-validation@1.19.1/dist/jquery.validate.min.js", at: "Foot", depends_on: "jquery" %}
```

Comment:
`depends_on` or `depends-on`?

Reply:
Never use a dash in liquid, you can see all parameter name in the `ScriptTag.cs` file of the OC.Resources project

Reply:
Seem we need to fix them in mny places then

Reply:
Have a look https://github.com/OrchardCMS/OrchardCore/search?q=depends-on

Reply:
Yes, we use `depends-on` in razor files when using our custom Script TagHelper

But here we need `depends_on` as we are in a Liquid file using our Liquid ScriptTag